task-36/src/components/Smile.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+class Smile extends React.Component {
+    constructor(props){
+        super(props);
+        this.state={
+            vol: props.initialValue
+        }
+    }
+
+    render() {
+        return (
+            <div className="block">
+                <span role="button" onClick={()=> {
+                    this.setState({vol:  this.state.vol += 1})
```

Comment:
```suggestion
                    this.setState({vol:  this.state.vol + 1})
```
в консолі є попередження, що ви змінюєте стейт безпосередньо